tests/resources/Util.robot

```diff
@@ -7,8 +7,6 @@ Library  Process
 
 *** Variables ***
 ${bin-dir}  /drone/src/github.com/vmware/vic/bin
-${vch-name}  VCH-default
-${install}  bin/vic-machine -name ${vch-name} -target=%{ESX_URL} -user=%{ESX_USERNAME} -image-store=datastore1 -appliance-iso=bin/appliance.iso -bootstrap-iso=bin/bootstrap.iso -generate-cert=true -passwd=%{ESX_PASSWORD} -force=true -bridge-network=network -compute-resource=%{ESX_RESOURCE}
 
 *** Keywords ***
 Install VIC Appliance To ESXi Server
```

Comment:
just to confirm that if this case will run on VC? If that's the case, you probably need to change this comment.